messages/es.json

```diff
@@ -399,6 +399,7 @@
   "admin/pages.editor.locale.za": "Zhuang",
   "admin/pages.editor.locale.zh": "Chino",
   "admin/pages.editor.locale.zu": "Zulú",
+  "admin/pages.editor.store.settings.advanced.disclaimer": "⚠️ Estas características deben usarse con atención.",
```

Comment:
Is this Vlad's translations?

Reply:
Nope, how should I get this into his backlog?